doc/calcurse.1.txt

```diff
@@ -30,519 +30,714 @@
+The following structure is created by default in your $HOME directory the
+first time calcurse is run without any options:
 
 ----
 $HOME/.calcurse/
-          |___notes/
-          |___conf 
+          |___apts
+          |___conf
+          |___hooks/
           |___keys
-          |___apts 
+          |___notes/
           |___todo
 ----
 
-The 'notes' subdirectory contains descriptions of the notes which are attached
-to appointments, events or todos. One text file is created per note, whose name
-is the SHA1 message digest of the note itself.
+The files are of two different kinds: data and configuration. The data files
+constitute the calcurse database and are independent of the calcurse release
+version; the configuration files depend on the calcurse version although
+backwards compatibility is always aimed at.
+
+.Data files
 
-The 'conf' file contains the user configuration. The 'keys' file contains the
-user-defined key bindings. The 'apts' file contains all of the user's
-appointments and events, and the 'todo' file contains the todo list.
+The calendar file +apts+ contains all of the user's appointments and events,
+and the +todo+ file contains the todo list.  The +notes+ subdirectory contains
+the notes which are attached to appointments, events or todos.  One text file
+is created per note, whose name is the SHA1 message digest of the note itself.
+
+The (hidden) lock files of the calcurse (+.calcurse.pid+) and daemon
+(+.daemon.log+) programs are present when they are running.  If daemon log
+activity has been enabled in the notification configuration menu, the file
++daemon.log+ is present.
+
+An alternative calendar file may be specified with the *-c* option.
+
+.Configuration files
+
+The +conf+ file contains the user configuration and the +keys+ file 
+the user-defined key bindings. The +hooks+ directory contains user-supplied
+scripts, see <<_hooks,Hooks>>.
+
+.Directory configuration
+
+An alternative directory to the default +$HOME/.calcurse+ may be specified
+with the *-D* option.
+
+An alternative directory for the configuration files 'only' may be specified
+with the *-C* option; in that case data files are either in the default
+directory or in the directory specified by *-D*. If both *-D* and
+*-C* are present, configuration files in the data directory, if any, are
+ignored.
+
+----
+<datadir>      <confdir>
+     |             |
+     |__ apts      |___ conf
+     |__ todo      |___ keys
+     |__ notes/    |___ hooks/
+
+default for both: $HOME/.calcurse/
+----
 
-'Note:' if the logging of calcurse daemon activity was set in the notification
-configuration menu, the extra file 'daemon.log' will appear in calcurse data
-directory. This file contains logs about calcurse activity when running in
-background.
+Calcurse may switch between two configuration setups, but still access
+the same data files e.g. with:
 
-Environment
+----
+$ calcurse
+
+$ calcurse -C $HOME/.calcurse/config
```

Comment:
This should be quoted: `calcurse -C "$HOME/.calcurse/config"`.

Reply:
If the home directory contains any space, the command without the quotes breaks. It's a corner case but it does not hurt to convert the example into a proper command that works under all circumstances.

Reply:
Corrected. Spaces crept into directory names a long time ago. Sigh.